src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  margin-top: 5px;
+}
+
+.header {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: black;
+}
+
+.nav-link {
+  margin-right: 20px;
+}
+
+.nav-link:last-child {
```

Comment:
You can use `.nav-link:not(:last-child)` selector to not apply margin-right to the last element, it shorthen a bit your code

Reply:
ok, thank you, changes are pushed,